include/glow/Base/Type.h

```diff
@@ -113,6 +113,39 @@ struct ShapeNCHW {
   }
 };
 
+struct ShapeTLBR {
+  /// Top padding.
+  size_t top;
+  /// Left padding.
+  size_t left;
+  /// Bottom padding.
+  size_t bottom;
+  /// Right padding.
+  size_t right;
+
+  explicit ShapeTLBR(llvm::ArrayRef<size_t> shape) {
+    assert(shape.size() == 4 && "Invalid padding");
+    top = shape[0];
+    left = shape[1];
+    bottom = shape[2];
+    right = shape[3];
+  }
+
+  static ShapeTLBR zero() { return ShapeTLBR(0, 0, 0, 0); }
```

Comment:
minor: when do we want to create an empty pads?

Reply:
Thought about default value. But now I think it is unnecessary :)